drivers/bus/Kconfig

```diff
@@ -242,4 +242,14 @@ config DA8XX_MSTPRI
 source "drivers/bus/fsl-mc/Kconfig"
 source "drivers/bus/mhi/Kconfig"
 
+config ANCILLARY_BUS
+       tristate "Software based Ancillary Bus"
+       help
+         Provides a software bus for ancillary_devices to be added to it
+         and ancillary_drivers to be registered on it.  It matches driver
+         and device based on id and calls the driver's pobe routine.
+         One example is the irdma driver needing to connect with various
+         PCI LAN drivers to request resources (queues) to be able to perform
+         its function.
```

Comment:
irdma driver is not in the kernel right? And this patchset is going up the sound-tree. So can you mention this example?
For the rdma driver attaching to the PCI device owned by different LAN driver, I would move it to an example in introduction or when to use section [dropping 'irdma' usage]. And give specific example for sound use-case here since that is the first consumer of the ancillary bus.

Reply:
Changed it to "another driver providing network based storage functionality"

Reply:
Missed this one - will do one more force push to get this change in